axon-server-connector/src/main/java/org/axonframework/axonserver/connector/AxonServerConnectionManager.java

```diff
@@ -555,11 +573,32 @@ public void onOutboundInstruction(String context,
      * @param instruction the message containing information for AxonServer to process
      */
     public void send(String context, PlatformInboundInstruction instruction) {
-        if (getChannel(context) != null) {
+        if (isConnected(context)) {
             instructionStreams.get(context).onNext(instruction);
         }
     }
 
+    /**
+     * Forces a disconnection from AxonServer for the specified context.
+     *
+     * @param context the (Bounded) Context for which the disconnection is required
+     * @param cause   the cause of the disconnection
+     */
+    public void disconnectExceptionally(String context, Throwable cause) {
+        if (isConnected(context)) {
+            instructionStreams.get(context).onError(cause);
+        }
+    }
+
+    /**
+     * Returns {@code true} if a gRPC channel for the specific context is opened between client and AxonServer.
```

Comment:
It might be worth to state that if no connection is currently alive, that it will attempt to set up a connection. I believe this is what the getChannel(context) does, correct?

Reply:
if no connection is currently alive, it is called the StreamObserver.onError that is responsible to reschedule a new connection with AxonServer.